generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -344,3 +346,29 @@ class FireSimTraceGenL2Config extends Config(
     outerLatencyCycles = 50) ++
   new WithTraceGenBridge ++
   new FireSimRocketChipConfig)
+
+
+class WithRationalTiles(multiplier: Int, divisor: Int) extends Config((site, here, up) => {
+  case FireSimClockKey => FireSimClockParameters(Seq(RationalClock("TileDomain", multiplier, divisor)))
+  case RocketCrossingKey => up(RocketCrossingKey, site) map { r =>
+    r.copy(crossingType = RationalCrossing())
+  }
+  case BoomCrossingKey => up(BoomCrossingKey, site) map { r =>
+    r.copy(crossingType = RationalCrossing())
+  }
```

Comment:
Hmm. I have no idea if the `ArianeTile` deals with crossings properly... but I think this should include this for now:

```
case ArianeCrossingKey => up(ArianeCrossingKey, site) map { a =>
    a.copy(crossingType = RationalCrossing())
  }
```

Reply:
Also nit: change the map from `r` -> `b` for BOOM?